### PR TITLE
docs: some steps to use gh pages

### PR DESCRIPTION
**Pull request by erickzanardo**
# Description

Some instructions on how to use github pages to deploy a web game.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors and are passing (See [Contributor Guide]).
- [ ] My PR does not decrease the code coverage, or I have __a very special case__ and explained on the PR description why this PR decreases the coverage.
- [ ] I updated/added relevant documentation (doc comments with `///`) and updated/added examples in `doc/examples`.
- [ ] I have formatted my code with `./scripts/format.sh` and the Flame analyzer (`./scripts/analyze.sh`) does not report any problems.
- [ ] I read and followed the [Flame Style Guide].
- [ ] I have added a description of the change under `[next]` in `CHANGELOG.md`.
- [ ] I removed the `Draft` status, by clicking on the `Ready for review` button in this PR.
- [ ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flame users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in `CHANGELOG.md`).
- [ ] No, this is *not* a breaking change.

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database]. Indicate, which of these issues are resolved or fixed by this PR.*

[issue database]: https://github.com/flame-engine/flame/issues
[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Flame Style Guide]: https://github.com/flame-engine/flame/blob/main/STYLEGUIDE.md

### CODE REVIEW

File: doc/platforms.md [hunk 1]
```diff
@@ -26,6 +26,54 @@ To build the game for production, using skia, use the following:
...
+
+Here we will explain the easiest way to get your game hosted using that.
+
+First thing, lets create the branch where your deployed files will live:
+
+```bash
+git checkout -b gh-pages
+```
+
+This branch be created from `main` or any other place, it doesn't matter much. After you push that
```
Comment (erickzanardo) [inline doc/platforms.md:42]:
```suggestion
This branch can be created from `main` or any other place, it doesn't matter much. After you push that
```

File: doc/platforms.md [hunk 2]
```diff
@@ -26,6 +26,54 @@ To build the game for production, using skia, use the following:
 $ flutter build web --release --web-renderer canvaskit
 ```
 
+## Deploy your game to GitHub Pages
+
+One easy easy way to deploy your game online, it to use [GitHub Pages])https://pages.github.com/).
```
Comment (spydon) [inline doc/platforms.md:31]:
```suggestion
One easy way to deploy your game online, is to use [GitHub Pages])https://pages.github.com/).
```

File: doc/platforms.md [hunk 3]
```diff
@@ -26,6 +26,54 @@ To build the game for production, using skia, use the following:
...
+It is a cool feature from GitHub, where you can easily host web content from your repository.
+
+Here we will explain the easiest way to get your game hosted using that.
```
Comment (spydon) [inline doc/platforms.md:34]:
```suggestion
Here we will explain the easiest way to get your game hosted using GitHub pages.
```

File: doc/platforms.md [hunk 4]
```diff
@@ -26,6 +26,54 @@ To build the game for production, using skia, use the following:
...
+      - uses: subosito/flutter-action@v1
+      - uses: bluefireteam/flutter-gh-pages@v7
+        with:
+          baseHref: /NAME_OF_YOUR_REPOSITORY/
+          webRenderer: canvaskit
+```
+
+Be sure to change `NAME_OF_YOUR_REPOSITORY` to the name of your GitHub repository.
+
+Now, whenever you commit something on the `main` branch, the action will run and updated your
```
Comment (spydon) [inline doc/platforms.md:71]:
```suggestion
Now, whenever you push something to the `main` branch, the action will run and update your
```

**Review by spydon — APPROVED** [review]
Nice, just some small comments

File: doc/platforms.md [hunk 5]
```diff
@@ -26,6 +26,54 @@ To build the game for production, using skia, use the following:
 $ flutter build web --release --web-renderer canvaskit
 ```
 
+## Deploy your game to GitHub Pages
+
+One easy way to deploy your game online, is to use [GitHub Pages])https://pages.github.com/).
```
Comment (luanpotter) [inline doc/platforms.md:31]:
```suggestion
One easy way to deploy your game online, is to use [GitHub Pages](https://pages.github.com/).
```

File: doc/platforms.md [hunk 6]
```diff
@@ -26,6 +26,54 @@ To build the game for production, using skia, use the following:
...
+It is a cool feature from GitHub, where you can easily host web content from your repository.
```
Comment (luanpotter) [inline doc/platforms.md:32]: > It is a cool feature from GitHub

You startled the ~witch~ @wolfenrain

Comment (luanpotter) [reply doc/platforms.md:32]: *by which

**Review by luanpotter — APPROVED** [review]
LGTM

File: doc/platforms.md (hunk 6 above)
Comment (erickzanardo) [reply doc/platforms.md:32]:
```suggestion
It is a cool feature from GitHub, by which you can easily host web content from your repository.
```

**Pull request merged**